manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -593,7 +626,7 @@ func (r *FybrikApplicationReconciler) updateProvisionedStorageStatus(application
 			return false, nil
 		}
 		if !res.Provisioned {
-			r.Log.V(0).Info("No bucket has been provisioned for " + id)
+			r.Log.Error().Err(errors.New(res.ErrorMsg)).Str(logging.FYBRIKAPPUUID, utils.GetFybrikApplicationUUID(applicationContext)).Str(logging.ACTION, logging.CREATE).Str(logging.DATASETID, id).Msg("No bucket has been provisioned for " + id)
```

Comment:
At this point we don't know if it is an error, the request can be in pending state.

Reply:
Changed to warn